src/test/streaming_test.cpp

```diff
@@ -527,18 +585,25 @@ node {
     // Invalid request - missing data in buffer
     prepareInvalidRequest(this->firstRequest, {"in"});  // no timestamp specified, server will assign one
 
-    EXPECT_CALL(this->stream, Read(_)).Times(0);
-    EXPECT_CALL(this->stream, Write(_, _)).Times(0);
+    std::mutex mtx;
+
+    EXPECT_CALL(this->stream, Read(_))
+        .WillOnce(DisconnectWhenNotified(mtx));
+    EXPECT_CALL(this->stream, Write(_, _))
+        .WillOnce(SendErrorAndNotifyEnd(
+            Status(StatusCode::INVALID_CONTENT_SIZE).string()
+                + std::string{" - Expected: 4 bytes; Actual: 0 bytes; input name: in; partial deserialization of first request"},
+            mtx));
 
-    ASSERT_EQ(executor.inferStream(this->firstRequest, this->stream), StatusCode::INVALID_CONTENT_SIZE);
+    ASSERT_EQ(executor.inferStream(this->firstRequest, this->stream), StatusCode::OK);
```

Comment:
So here the inferStream call returns OK despite single request being denied. We should have such scenarios in functional tests, so we make sure we have the correct behavior on the user side.

Reply:
@jasiu86